test/autoprefixer.test.js

```diff
@@ -144,7 +144,7 @@ const COMMONS = [
     'advanced-filter', 'element', 'image-set', 'image-rendering',
     'mask-border', 'writing-mode', 'cross-fade', 'gradient-fix',
     'text-emphasis-position', 'grid', 'grid-area', 'grid-template',
-    'grid-template-areas'
+    'grid-template-areas', 'color-adjust'
```

Comment:
Should I add 'color-adjust' declaration to the array of `COMMONS`? Not sure about this change. It was a little hard to get the whole structure and logic inside this test file.

Reply:
Yes, you should add it to `COMMONS` to run `color-adjust` with not-duplication and prefixes removing tests